test/Distributed/SIL/distributed_actor_remoteCall_sil.swift

```diff
@@ -15,91 +15,94 @@ distributed actor MyDistActor {
 
 // TODO(distributed): very naive test, SIL for discussion and cleanup
 
-// CHECK: // MyDistActor.test()
+// MyDistActor.test()
```

Comment:
Sorry that the test is so annoying, I will make it proper as the TODO mentions above